pkg/kv/kvserver/replica_init.go

```diff
@@ -128,7 +129,14 @@ func newUnloadedReplica(
 	r.splitQueueThrottle = util.Every(splitQueueThrottleDuration)
 	r.mergeQueueThrottle = util.Every(mergeQueueThrottleDuration)
 
-	r.breaker = newReplicaCircuitBreaker(store.cfg.Settings, store.stopper, r.AmbientContext, r)
+	r.breaker = newReplicaCircuitBreaker(
+		store.cfg.Settings, store.stopper, r.AmbientContext, r, func() {
+			telemetry.Inc(telemetryTripAsync)
+			r.store.Metrics().ReplicaCircuitBreakerCumTripped.Inc(1)
+			store.Metrics().ReplicaCircuitBreakerCurTripped.Inc(1)
+		}, func() {
+			store.Metrics().ReplicaCircuitBreakerCurTripped.Dec(1)
+		})
```

Comment:
Consider either naming these closures, or adding a comment to clarify their purpose.